### ISSUE TITLE
Fancy Place Animation with Torches and other Non-Solid Blocks

### ISSUE THREAD

**Issue by razorrider7** [TominoCZ/FancyBlockParticles#71]
## Steps to reproduce issue:
Turn on "Fancy Place Animation" and placed blocks, such as torches, will move you out of the way.
## MC Version:
1.12
## FBP Version:
2.3.9
## Forge Version (e.g. 2604):
2654

**Comment by TominoCZ** [comment]
Yeah... I know.. I'm really not sure if I'll be able to fix this, but I'll try one I have time to do it

**Comment by TominoCZ** [comment]
Will be fixed in the next version :)
